UnitTests/Services/Test_BingService.cs

```diff
@@ -15,15 +15,30 @@ public class Test_BingService
 
         [TestCategory("Services")]
         [TestMethod]
-        public async Task Test_BingService_Request()
+        public async Task Test_BingServiceSearch_Request()
         {
             BingSearchConfig config = new BingSearchConfig();
             config.Country = BingCountry.France;
+            config.QueryType = BingQueryType.Search;
             config.Query = Query;
 
             var results = await BingService.Instance.RequestAsync(config, 50);
 
             Assert.AreEqual(results.Count, 50);
         }
+
+        [TestCategory("Services")]
+        [TestMethod]
+        public async Task Test_BingServiceNews_Request()
+        {
+            BingSearchConfig config = new BingSearchConfig();
+            config.Country = BingCountry.France;
+            config.QueryType = BingQueryType.News;
+            config.Query = Query;
+
+            var results = await BingService.Instance.RequestAsync(config, 50);
+
+            Assert.AreEqual(results.Count, 10);
```

Comment:
With the news request, only max 10 results are returned.
The "count" parameter is not taken into consideration. :/